src/text.rs

```diff
@@ -651,7 +652,7 @@ impl TextCache {
                             .metadata(section.index),
                     )
                 }
-                let buffer_line = BufferLine::new(line_str, attrs_list);
+                let buffer_line = BufferLine::new(line_str, attrs_list, Shaping::Basic);
```

Comment:
```suggestion
                let buffer_line = BufferLine::new(line_str, attrs_list, Shaping::Advanced);
```

It looks like advanced shaping is what includes font fallback, so we'll want that (an example to check is the :eyes: emoji in the `README.md` since emojis are usually only included in a handful of different fonts)

Reply:
I wasn't sure about this one. It renders correctly with `Basic` for me. I guess that means my system font supports emojis...

Reply:
Ah, it doesn't for me, so easier for me to spot 😅 (Eventually I'd like to get testing setup where we can render the output to something like a PNG and use an embedded set of fonts so that we can actually programmatically check these things)

Reply:
That's be nice yeah. Even if it means using a shell script or some external tool that'd be better than nothing.